spec.md

```diff
@@ -148,20 +148,47 @@ specific protocols (AWS Kinesis, Azure Event Grid).
 
 ## Type System
 
-The following abstract data types are available for use in attributes.
+The following abstract data types are available for use in attributes. Each of
+these types MAY be represented differently by different event formats and in
+transport metadata fields. This specification defines a canonical string-encoding
+for each type that MUST be supported by all implementations.
```

Comment:
Even if that impl only supports transports/formats that don't use the string encoding?

Reply:
@duglin Yes, all implementations need to expect a string value to come across the wire where the "native" representation would be a binary-encoded number, for instance. Interop.